gax/src/main/java/com/google/api/gax/rpc/ServerStreamingAttemptCallable.java

```diff
@@ -175,7 +175,9 @@ public void cancel() {
         });
 
     if (autoFlowControl) {
-      pendingRequests = Integer.MAX_VALUE;
+      synchronized (lock) {
```

Comment:
Is this a trivial change? Is it correct? It took me a while to figure out what pendingRequests was doing when it was set to MAX_VALUE. It looks like a trivial change to me since it is in the start() method, so there should be no contention here anyway. Is that right?

Reply:
It should be trivial, I also consulted with igorbernstein (the author of the file).